app/src/components/Charts/StreamPerHour/query.tsx

```diff
@@ -1,7 +1,8 @@
 import { TABLE } from '../../../db/queries/constants'
 import type { Int } from 'apache-arrow'
 
-export const query = `
+export function query(year: number | undefined) {
```

Comment:
Maybe it would be easier to understand what this query is about with another name than `query` (when importing in another component for example)